### PR TITLE
Add helper functions for delta TS to significance conversion

### PR DESCRIPTION
**Pull request by QRemy**
Add helper functions for delta TS to significance conversion.
Splitted from PR #4216.

### CODE REVIEW

**Review by AtreyeeS — APPROVED** [review]
Thanks @QRemy !
I think this will be very useful to users, so maybe it is worth expanding the doc strings to include the `Parameters` and specify this in the `Compute Source Significance` **HowTo**

File: gammapy/stats/utils.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+from scipy.stats import chi2
+
+
+def sigma_to_ts(n_sigma, df=1):
+    """Convert sigma to delta ts"""
```
Comment (registerrier) [inline gammapy/stats/utils.py:8]:
```suggestion
    """Convert number of sigma to delta ts."""
```

File: gammapy/stats/utils.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    p_value = chi2.sf(n_sigma**2, df=1)
+    return chi2.isf(p_value, df=df)
+
+
+def ts_to_sigma(ts, df=1):
+    """Convert delta ts to sigma"""
```
Comment (registerrier) [inline gammapy/stats/utils.py:14]:
```suggestion
    """Convert delta ts to number of sigma"""
```

File: gammapy/stats/utils.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+from scipy.stats import chi2
```
Comment (registerrier) [inline gammapy/stats/utils.py:4]:
```suggestion
from scipy.stats import chi2, norm
```

File: gammapy/stats/utils.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+
+def sigma_to_ts(n_sigma, df=1):
+    """Convert sigma to delta ts"""
+    p_value = chi2.sf(n_sigma**2, df=1)
```
Comment (registerrier) [inline gammapy/stats/utils.py:9]:
```suggestion
    p_value = norm.sf(n_sigma)
```

File: gammapy/stats/utils.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    return chi2.isf(p_value, df=df)
+
+
+def ts_to_sigma(ts, df=1):
+    """Convert delta ts to sigma"""
+    p_value = chi2.sf(ts, df=df)
+    return np.sqrt(chi2.isf(p_value, df=1))
```
Comment (registerrier) [inline gammapy/stats/utils.py:16]:
```suggestion
    return norm.isf(0.5*p_value, df=1)
```

**Review by adonath — COMMENTED** [review]
Thanks @QRemy! I guess we should adapt the docs here https://docs.gammapy.org/1.0/user-guide/stats/index.html#estimating-ts

**Review by registerrier — APPROVED** [review]
Thanks @QRemy. Looks good to me.

**Review by AtreyeeS — APPROVED** [review]
Thanks @QRemy ! The docs modification may also done in a follow up PR if you prefer...

**Pull request merged**